src/Microsoft.ML.TensorFlow/TensorflowTransform.cs

```diff
@@ -837,7 +838,7 @@ private Tensor CastDataAndReturnAsTensor(T[] data)
                         bytes[i] = Encoding.UTF8.GetBytes(((ReadOnlyMemory<char>)(object)data[i]).ToArray());
                     }
 
-                    return new Tensor(bytes, _tfShape.dims.Select(x => (long)x).ToArray());
+                    return new Tensor(new NDArray(bytes, _tfShape));
```

Comment:
@Oceania2018 
the type of bytes here is byte[][], Can you confirm it is the right way to create a 2-d byte tensor.

Reply:
You can use `new Tensor(byte[][])` if shape is `shape(x, y)`